Taarafo.Core.Tests.Unit/Services/Foundations/Posts/PostServiceTests.Logic.RetrieveById.cs

```diff
@@ -0,0 +1,51 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Force.DeepCloner;
+using Moq;
+using Taarafo.Core.Models.Posts;
+using Xunit;
+
+namespace Taarafo.Core.Tests.Unit.Services.Foundations.Posts
+{
+    public partial class PostServiceTests
+    {
+        [Fact]
+        public async Task ShouldRetrievePostByIdAsync()
+        {
+            // given
+            Post randomPosts = CreateRandomPost();
+            Guid inputPostId = Guid.NewGuid();
+            Post storagePosts = randomPosts;
+            Post expectedPosts = storagePosts.DeepClone();
+
+            this.storageBrokerMock.Setup(broker =>
+                broker.SelectPostByIdAsync(inputPostId))
+                    .ReturnsAsync(storagePosts);
+
+            // when
+            Post actualPosts =
+                await this.postService.RetrievePostByIdAsync(inputPostId);
+
+            // then
+            actualPosts.Should().BeEquivalentTo(expectedPosts);
+
+            this.dateTimeBrokerMock.Verify(broker =>
+                broker.GetCurrentDateTimeOffset(),
+                    Times.Never);
```

Comment:
You don't need to verify this is never called.
Because of two reasons:
1. It's not critical business path.
2. It will never be called anyway. So `VerifyNotOtherCalls` will take care of it.